packages/xo-server-backup-reports/src/index.js

```diff
@@ -154,6 +154,10 @@ class BackupReportsXoPlugin {
   }
 
   _wrapper(status, job, schedule, runJobId) {
+    if (job.type === 'metadataBackup') {
+      return
+    }
```

Comment:
Why?

Reply:
I will do the implementation of the report in an other PR.